client/src/components/CredentialsInputBox/CredentialsInputBox.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
It looks like the old code got merged in here and wasn't removed so we can remove the CredentialsInputBox folder.
The code has been moved in the `InputBox` and `CredentialsForm` folders

Reply:
Removed from current branch